packages/renderer/src/lib/window-control-buttons/WindowsControlButton.svelte

```diff
@@ -58,10 +58,10 @@ function executeAction() {
   on:click={() => executeAction()}
   aria-label={name}
   title={titleName}
-  class="h-[32px] w-[45px] cursor-pointer text-white {name === 'Close'
+  class="h-[32px] w-[45px] cursor-pointer {name === 'Close'
     ? 'hover:bg-[#be4425]'
-    : 'hover:bg-[#2d2d2d]'} flex place-items-center justify-center">
+    : 'hover:bg-charcoal-300'} flex place-items-center justify-center">
```

Comment:
why charcoal-300 and not 2d2d2d ?

Reply:
@benoitf That's what Linux uses, I've discussed this with @ekidneyrh, and she will create a ticket for herself to change colors to be united and slightly redesign the topbar in the next sprint

Reply:
AFAIK the colors were picking up the default colors of Windows so the bar was looking the same than the native widget of users.

Reply:
Ok, I'll keep the 2d2d2d then

Reply:
well I think you should use a variable, but in color-registry maybe we should override the value in case of Windows 

if (isWindows()) {
do not use charcoal but like 2d2d
and handle the light and dark colors
}

Reply:
Well this is the Windows (Linux is has own implementation [here](https://github.com/containers/podman-desktop/blob/main/packages/renderer/src/lib/window-control-buttons/LinuxControlButton.svelte) and Mac is native? since I don't see the implementation in repo).

So I think adding this would be probably unnecessary?

Reply:
no color definition should happen outside of the color-registry

the reason is that if you use a variable, it's possible to override the value by an extension or other mechanism while if it's hardcoded in the svelte file, you need to recompile the binary

Reply:
Ok, so I will add be4425 and 2d2d2d to color-registry and use the variables in here

Reply:
Should I add those colors to `tailwind-color-palette.json` and from there use them in color-registry?

Reply:
I would say no as it's not part of the command palette